_rules/auto-play-audio-does-not-exceed-3-seconds-aaa1bf.md

```diff
@@ -1,9 +1,9 @@
 ---
 id: aaa1bf
-name: auto-play audio does not exceed 3 seconds
+name: '`autoplay` audio does not exceed 3 seconds'
 rule_type: atomic
 description: |
-  auto-play audio content does not output audio for more than 3 seconds
+  '`autoplay` audio content does not output audio for more than 3 seconds'
```

Comment:
Same as my previous comment.

Suggestion: Autoplay audio content -> Audio content that plays automatically